src/index.ts

```diff
@@ -6,7 +7,23 @@ const createTransformer = (options?: any): jest.Transformer => {
   return {
     canInstrument: true,
     getCacheKey,
-    process: preprocess,
+    process: (
+      sourceText: string,
+      sourcePath: string,
+      config: jest.ProjectConfig,
+      options?: jest.TransformOptions
+    ) => {
+      try {
+        return preprocess(
+          sourceText,
+          sourcePath,
+          config,
+          options
+        );
+      } finally {
+        flushLogs();
```

Comment:
I don't think the finally will be ran since there is a `return` in the `try` block, is it?

Reply:
As specified [here](https://tc39.github.io/ecma262/#sec-try-statement-runtime-semantics-evaluation), the `finally` block is always executed regardless of what happens in the `try` or `catch` blocks. I also tested it manually. :)

Reply:
Cool! Thanks, I didn't know and learned something here!